slowish/models.py

```diff
@@ -75,3 +75,21 @@ class Meta:
 
     def __unicode__(self):
         return "{0} (in account {1})".format(self.name, self.account.id)
+
+
+class SlowishFile(models.Model):
+    """A file, (within a particular container)."""
+
+    container = models.ForeignKey(SlowishContainer, on_delete=models.CASCADE)
+
+    path = models.CharField(
+        help_text="Complete path of this file (within the container).",
+        max_length=255)
```

Comment:
The restriction of length=255 for unique=True is only for MySQL, correct? Can we ignore that?

I don't have unique=True set directly on the path, but I do have unique_together=('container', 'path') and I don't know how that interacts with this limitation here.

Meanwhile, Slowish is only currently intended to support the needs of the repository-swift test suite, and I don't expect it to start getting anywhere close to either 255 nor 1024.

I think I will just set this to max_length=1024 since that makes things less likely to ever cause a problem, (and I'm hoping that if someone does run into the MySQL unique-no-longer-than-255 limit because of this that it will at least complain noisily upfront rather than failing in some mysterious way).